base/ltfssaxes.dtx

```diff
@@ -1260,77 +4035,220 @@
 %    faster, but such a  setup would be so weird in the first place
 %    that this isn't worth the effort.
-%
 %    \begin{macrocode}
-%\DeclareFontShapeChangeRule {scsl}{n}   {n}     {}
+\DeclareFontShapeChangeRule {scit}{up}  {sc}    {}
+\DeclareFontShapeChangeRule {scit}{ssc} {sscit} {scit}
+%    \end{macrocode}
+%    
+%    \begin{macrocode}
+% \DeclareFontShapeChangeRule {scsl}{n}   {n}     {}
 \DeclareFontShapeChangeRule {scsl}{it}  {scit}  {scsl}
 \DeclareFontShapeChangeRule {scsl}{sl}  {scsl}  {}
-\DeclareFontShapeChangeRule {scsl}{sw}  {scsw}  {sc}    % or scsl?
+\DeclareFontShapeChangeRule {scsl}{sw}  {scsw}  {scsl}
 \DeclareFontShapeChangeRule {scsl}{sc}  {scsl}  {}
 \DeclareFontShapeChangeRule {scsl}{ulc} {sl}    {}
-\DeclareFontShapeChangeRule {scsl}{up}   {sc}   {}
+\DeclareFontShapeChangeRule {scsl}{up}  {sc}    {}
+\DeclareFontShapeChangeRule {scsl}{ssc} {sscsl} {scsl}
 %    \end{macrocode}
 %
 %    \begin{macrocode}
-%\DeclareFontShapeChangeRule {scsw}{n}   {n}     {}
+% \DeclareFontShapeChangeRule {scsw}{n}   {n}     {}
 \DeclareFontShapeChangeRule {scsw}{it}  {scit}  {scsw}
 \DeclareFontShapeChangeRule {scsw}{sl}  {scsl}  {}
 \DeclareFontShapeChangeRule {scsw}{sw}  {scsw}  {}
 \DeclareFontShapeChangeRule {scsw}{sc}  {scsw}  {}
 \DeclareFontShapeChangeRule {scsw}{ulc} {sw}    {}
-\DeclareFontShapeChangeRule {scsw}{up} {sc}     {}
+\DeclareFontShapeChangeRule {scsw}{up}  {sc}    {}
+\DeclareFontShapeChangeRule {scsw}{ssc} {sscsw} {scsw}
 %    \end{macrocode}
 %
 %    \begin{macrocode}
-%\DeclareFontShapeChangeRule {sw}{n}   {n}     {}
-%\DeclareFontShapeChangeRule {sw}{it}  {it}    {}
-%\DeclareFontShapeChangeRule {sw}{sl}  {sl}    {}
-%\DeclareFontShapeChangeRule {sw}{sw}  {sw}    {}
-\DeclareFontShapeChangeRule {sw}{sc}  {scsw}  {}
+% \DeclareFontShapeChangeRule {sw}{n}   {n}     {}
+% \DeclareFontShapeChangeRule {sw}{it}  {it}    {}
+% \DeclareFontShapeChangeRule {sw}{sl}  {sl}    {}
+% \DeclareFontShapeChangeRule {sw}{sw}  {sw}    {}
+\DeclareFontShapeChangeRule {sw}{sc}  {scsw}  {scit}
 \DeclareFontShapeChangeRule {sw}{ulc} {sw}    {}
-\DeclareFontShapeChangeRule {sw}{up} {n}      {}
+\DeclareFontShapeChangeRule {sw}{up}  {n}     {}
+\DeclareFontShapeChangeRule {sw}{ssc} {sscsw} {scsw}
+%    \end{macrocode}
+%    
+%    \begin{macrocode}
+% \DeclareFontShapeChangeRule {ssc}{n}   {n}      {}
+\DeclareFontShapeChangeRule {ssc}{it}  {sscit}  {sscsl}
+\DeclareFontShapeChangeRule {ssc}{sl}  {sscsl}  {sscit}
+\DeclareFontShapeChangeRule {ssc}{sw}  {sscsw}  {sscit}
+\DeclareFontShapeChangeRule {ssc}{sc}  {sc}     {}
+% \DeclareFontShapeChangeRule {ssc}{ssc}  {ssc}   {}
+\DeclareFontShapeChangeRule {ssc}{ulc} {n}      {}
+%    \end{macrocode}
+%    We implement the same logic as for \texttt{sc}, see above. The
+%    \texttt{ssc} shape doesn 't to care for 30 years of history, but
+%    it would be surprising is \cs{sscshape}\cs{upshape} would work
+%    differently to \cs{scshape}\cs{upshape}.
```

Comment:
```suggestion
%    \texttt{ssc} shape doesn't need to care for 30 years of history,
%    but it would be surprising is \cs{sscshape}\cs{upshape} would
%    work differently to \cs{scshape}\cs{upshape}.
```

Reply:
[pedantry]

differently to -> differently from

Reply:
And (not pedantry): is ==> if 

Maybe pedantry: 

. . . would work . . . 
  ==> 
. . . were to work in a way that differs from that of . . .